src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -529,7 +529,7 @@ public void PublishUnwindInfo(ObjectNode node)
                 byte[] blobSymbolName = _sb.Append(_currentNodeZeroTerminatedName).ToUtf8String().UnderlyingArray;
 
                 EmitAlignment(4);
-                EmitSymbolDef(blobSymbolName);
+                EmitSymbolDef(blobSymbolName, false);
```

Comment:
I know this is my code, but could you make these `global: false` so that it's clear what the `false` stands for?

Maybe we could also make `false` the default parameter for this to reduce clutter.